libs/sdk-ui-ext/src/internal/dashboardEmbedding/DashboardView/types.ts

```diff
@@ -59,6 +62,35 @@ export interface IDashboardViewConfig {
     locale?: ILocale;
 }
 
+/**
+ * Information about the DrillDown interaction - the attribute that is next in the drill down hierarchy.
+ */
+export interface IDrillDownDefinition {
+    type: "drillDown";
+    target: ObjRef;
+}
+
+/**
+ * Type-guard testing whether the provided object is an instance of {@link IDrillDownDefinition}.
+ * @alpha
+ */
+export function isDrillDownDefinition(obj: unknown): obj is IDrillDownDefinition {
+    return !isEmpty(obj) && (obj as IDrillDownDefinition).type === "drillDown";
+}
+
+/**
+ * A {@link IDrillEvent} with added field that contains info about all the drilling interactions set in KPI dashboards
+ * that are relevant to the given drill event (including drill downs).
+ */
+export interface IDashboardDrillEvent extends IDrillEvent {
+    drillDefinitions?: Array<DrillDefinition | IDrillDownDefinition>;
+}
+
+/**
+ * Callback called when a drill event occurs.
+ */
+export type OnFiredDashboardViewDrillEvent = (event: IDashboardDrillEvent) => ReturnType<OnFiredDrillEvent>;
```

Comment:
do these new types need api annotations?

Reply:
We will see, I will do a complete API review in a separate PR when taking DahboardView out of internal today.